junto-rust/zomes/core/code/src/expressions/definitions/group_entry_definitions.rs

```diff
@@ -0,0 +1,149 @@
+//Entry Definition(s)
+use super::app_definitions;
+
+use hdk::{
+    entry_definition::ValidatingEntryType,
+    holochain_core_types::{
+        dna::entry_types::Sharing,
+        cas::content::Address
+    }
+};
+
+pub fn group_definition() -> ValidatingEntryType {
+    entry!(
+        name: "group",
+        description: "Group Object Entry",
+        sharing: Sharing::Public,
+        //native_type: app_definitions::Group,
+        validation_package: || {
+            hdk::ValidationPackageDefinition::Entry
+        },
+
+        validation: |validation_data: hdk::EntryValidationData<app_definitions::Group>| {
+            Ok(())
+        },
+
+        links: [
+            from!(
+                "user",
+                tag: "pack", //Any tag or expression tag
+
+                validation_package: || {
+                    hdk::ValidationPackageDefinition::ChainFull
```

Comment:
You should use `hdk::ValidationPackageDefinition::Entry` instead of `hdk::ValidationPackageDefinition::ChainFull` unless you really need the entire authoring agents chain for validation. This can be a lot of data to transport across the network unnecessarily so definitely make sure you change this everywhere you can